src/App.js

```diff
@@ -9,20 +14,50 @@ import './App.css';
 import List from './pages/List/List';
 import AddItem from './pages/AddItem/AddItem';
 import NotFound from './pages/NotFound/NotFound';
+import Home from './pages/Home/Home';
 
 // Components
 import NavMenu from './components/NavMenu/NavMenu';
 
+// Functions
+import getToken from './lib/tokens';
+
 function App() {
+  const [userToken, setUserToken] = useState('');
+
+  useEffect(() => {
+    const token = localStorage.getItem('token');
+    if (token) {
+      setUserToken(token);
+    }
+  }, []);
+
+  function saveToken(e) {
+    e.preventDefault();
+    const token = getToken();
+    localStorage.setItem('token', token);
+    setUserToken(token);
+  }
   return (
     <Router>
       <div className="App container">
         <Switch>
           <Route exact path="/">
-            <List />
+            {userToken ? (
+              <Redirect to="/list" />
+            ) : (
+              <Home
+                userToken={userToken}
+                setUserToken={setUserToken}
```

Comment:
This prop no longer needs to be passed to `Home` as you aren't using it in that component any longer. 
```suggestion
```